src/Host/API/Impl/RSessionCallback.cs

```diff
@@ -51,5 +51,7 @@ public Task<MessageButtons> ShowMessageAsync(string message, MessageButtons butt
         public Task ViewFile(string fileName, string tabName, bool deleteFile, CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
         public Task ViewLibraryAsync(CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
         public Task ViewObjectAsync(string expression, string title, CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
+
+        public string GetLocalizedString(string id) => null;
```

Comment:
We can, but then it won't reflect culture changes until the session is restarted.